src/main/java/uk/gov/ida/eventemitter/EventEncrypter.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.ida.eventemitter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.Inject;
+import org.apache.commons.codec.binary.Base64;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+import java.security.SecureRandom;
+
+public class EventEncrypter implements Encrypter {
+
+    private static final int IV_SIZE = 16;
+    private final byte[] key;
+    private final ObjectMapper mapper;
+
+    @Inject
+    public EventEncrypter(final byte[] key,
+                          final ObjectMapper mapper) {
+        this.key = key;
+        this.mapper = mapper;
+    }
+
+    public String encrypt(final Event event) throws Exception {
+        final byte[] iv = generateIV();
```

Comment:
Similar to above - its wouldn't be obvious to me what `iv` is unless I'd been reading the docs with you last week. `initializationVector` would give me a bit more of an idea what was going on.